data/BattleCafe/overview.md

```diff
@@ -1,4 +1,66 @@
-### Description
-Battle Café is used to evolve Micery into the different forms of Alcremie.
-You have a limited amount of spins each day, and you get form based on what sweet you use,
-how long you spin, what direction and what time of the day.
```

Comment:
Was this supposed to be removed in this PR?

Reply:
So I initially hated that tiny blurb because it wasn't very informative, and it had weird grammatical errors. So I intended on remedying it and then totally forgot and never got back to it because I was so enthralled in table-making 😅😂 [a2c7445](https://github.com/pokeclicker/pokeclicker-wiki/pull/6/commits/a2c7445ebb024f690cfbfd76ec6a1d5a006b056a) adds in the missing information and then some!